apps/api/src/app/controllers/routes.controller.ts

```diff
@@ -0,0 +1,47 @@
+import { Controller, Get, Route, Post, Body, Put, Delete } from 'tsoa';
+import { RouteDto } from '@mogilev-guide/models';
+import { Inject } from '@mogilev-guide/api/ioc';
+import { RoutesService } from '@mogilev-guide/api/services/routes';
+import { RoutesConverter } from '@mogilev-guide/api/helpers';
+
+@Route('routes')
+export class RoutesController extends Controller {
+  @Inject() private RoutesService!: RoutesService;
+  @Inject() private RoutesConverter!: RoutesConverter;
```

Comment:
It would be better to use an interface rather than a specific type (dependency inversion principle)

Reply:
DI is realy hard to use with intefaces. This is topic for investigation.